tools/cvm-image-rewriter/README.md

```diff
@@ -124,6 +124,36 @@ Required
   -i <guest image>          Specify initial guest image file
 ```
 
+### 2.3 Run in Nested VM
+
+This tool can be run in a guest VM on the host, in case that users need to make the host clean.  
```

Comment:
Is it better to replace "make the host clean" with "prepare a clean environment".